src/engines/kotor/gui/widgets/kotorwidget.cpp

```diff
@@ -328,6 +330,10 @@ void KotORWidget::load(const Aurora::GFF3Struct &gff) {
 
 	if (border.fill.empty())
 		_quad->setColor(0.0f, 0.0f, 0.0f, 0.0f);
+	else if (border.r == 0 && border.g == 0 && border.b == 0)
+		_quad->setColor(1.0f, 1.0f, 1.0f, 1.0f);
```

Comment:
That's a bit...weird. So if the file specifies black, it's actually not?

Reply:
Would it be better if i add a border.hasColor Flag?

Reply:
Hmm, we need a better way to distinguish no between "no color defined" and "color defined, but it's black" here.

`KotORWidget::createBorder()` (which is the method filling in the border struct, yes?) should be extended. `struct Border` should get a `bool hasColor`, set to the return value of `b.hasField("COLOR")`.